client/devdocs/docs-selectors/param-type.jsx

```diff
@@ -2,30 +2,50 @@
  * External dependencies
  */
 import React, { PropTypes } from 'react';
-import { get } from 'lodash';
+import { get, union } from 'lodash';
 
-/**
- * Matches an expression type
- *
- * @type {RegExp}
- */
-const REGEXP_EXPRESSION_TYPE = /(.*)Type$/;
+const ARRAY_TYPE_PATTERN = /^Array[.]<([^>]+)>/;
```

Comment:
Are these normalized by `jsdoc-api`? Otherwise this might not capture some of its variants like `String[]`

http://usejsdoc.org/tags-type.html#jsdoc-types